effectful/effectful.cabal

```diff
@@ -76,14 +76,18 @@ library
 
     hs-source-dirs:  src
 
-    exposed-modules: Effectful.Concurrent
+    exposed-modules: Effectful.ByteString.Static
+                     Effectful.ByteString.Builder.Static
```

Comment:
These modules have misleading names. It should be something like `Effectful.Filesystem.IO.ByteString.Static` etc.